### ISSUE TITLE
Linter is broken with stylint >1.4.1

### ISSUE THREAD

**Issue by zenopopovici** [AtomLinter/linter-stylint#27]
```
Error: module.js:340 throw err; ^

Error: Cannot find module '//package.json' at Function.Module._resolveFilename (module.js:338:15) at Function.Module._load (module.js:289:25) at Module.require (module.js:366:17) at require (module.js:385:17) at Object. (/usr/local/lib/node_modules/stylint/index.js:31:14) at /usr/local/lib/node_modules/stylint/node_modules/stampit/stampit.js:91:25 at forEach (/usr/local/lib/node_modules/stylint/node_modules/mout/array/forEach.js:15:27) at Function.factory [as create] (/usr/local/lib/node_modules/stylint/node_modules/stampit/stampit.js:89:7) at Object. (/usr/local/lib/node_modules/stylint/bin/stylint:42:11) at Module._compile (module.js:425:26)
Error: module.js:340
    throw err;
    ^
Error: Cannot find module '//package.json'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:289:25)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Object.<anonymous> (/usr/local/lib/node_modules/stylint/index.js:31:14)
    at /usr/local/lib/node_modules/stylint/node_modules/stampit/stampit.js:91:25
    at forEach (/usr/local/lib/node_modules/stylint/node_modules/mout/array/forEach.js:15:27)
    at Function.factory [as create] (/usr/local/lib/node_modules/stylint/node_modules/stampit/stampit.js:89:7)
    at Object.<anonymous> (/usr/local/lib/node_modules/stylint/bin/stylint:42:11)
    at Module._compile (module.js:425:26)
    at parameters.exit (/Users/Zeno/.atom/packages/linter-jshint/node_modules/atom-linter/lib/helpers.js:47:27)
    at triggerExitCallback (/Applications/Atom.app/Contents/Resources/app.asar/src/buffered-process.js:215:47)
    at ChildProcess.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/buffered-process.js:237:18)
    at emitTwo (events.js:87:13)
    at ChildProcess.emit (events.js:172:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)
```

**Comment by fdeneux** [comment]
+1 Having the same issue.

**Comment by zenopopovici** [comment]
I think it has to do with the path ...

**Comment by Arcanemagus** [comment]
Looks like [this commit](https://github.com/rossPatton/stylint/commit/023996a60d6b56e3e911563da2bdd78f134f4aca) is what broke it, specifically [this line](https://github.com/rossPatton/stylint/commit/023996a60d6b56e3e911563da2bdd78f134f4aca#diff-168726dbe96b3ce427e7fedce31bb0bcR29) doesn't seem to be building a path properly.

@rossPatton it looks like that code assumes that the linter is being ran from the directory of the file being linted. This package doesn't actually set the `cwd` though. For most systems that ends up being the Atom folder but it looks like on some it may be getting the drive root, ending up with the `//package.json` seen above.

I can set the `cwd` here which should work around this bug, but you might want to look into why that breaks in the first place. Also, it would help when linking to your codebase if you had the releases tagged so specific versions of the code could easily be referenced. I use [this script](https://gist.github.com/Arcanemagus/6aa80a3cb63b1e188e91) to tag old releases with the proper timestamp, it shouldn't be too hard to adopt it to your OS.

**Comment by Arcanemagus** [comment]
@zenopopovici and @fdeneux what OS and Atom version are you on? The CI tests here didn't show a failure with `stylint@1.5.3`, so we obviously are missing something in the CI testing environment.

**Comment by Arcanemagus** [comment]
Nevermind... it looks like it wasn't running CI on that for some reason. It's completely broken even with the changes in #28 since there _must_ be a `package.json` in the same directory as the file or it fails.

**Comment by zenopopovici** [comment]
If you still need it:

```
Atom    : 1.8.0
Electron: 0.36.8
Chrome  : 47.0.2526.110
Node    : 5.1.1
```

**Comment by Arcanemagus** [comment]
v2.1.3 is out which leaves the package in a working state since it pins it to `stylint@1.3.10`. I left in the wcd fix since it should be in there anyway, but I'm not sure a version past that can be released yet since it seems v1.4.1 and up _require_ a `package.json` file in the same directory as the cwd.

Leaving this open for now since Greenkeeper seems to be having issues checking new versions.

**Comment by rossPatton** [comment]
this should be fixed in 1.5.4

**Comment by zenopopovici** [comment]
I confirm stylint is now fixed. However the atom plugin is now silent to all errors.

**Comment by Arcanemagus** [comment]
v2.2.0 is released updating the built in `stylint` to v1.5.4 and fixing the regex parsing the output, enjoy! 🎉
